cmd/sync/sync.go

```diff
@@ -199,22 +199,7 @@ func New() *cobra.Command {
 		var opts *sync.SyncOptions
 		var err error
 
-		//
-		// To be uncommented and used once our VS Code extension is bundle aware.
-		// Until then, this could interfere with extension usage where a `databricks.yml` file is present.
-		// See https://github.com/databricks/cli/pull/207.
-		//
-		// b := bundle.GetOrNil(cmd.Context())
```

Comment:
Can we complete get rid of bundle/context functionality or is there a strong use case for it still?